services/galley/test/integration/API/Teams.hs

```diff
@@ -123,14 +121,11 @@ tests s =
         [test s "the list should be truncated" testUncheckedListTeamMembers],
       test s "enable/disable SSO" testEnableSSOPerTeam,
       test s "enable/disable Custom Search Visibility" testEnableTeamSearchVisibilityPerTeam,
-      test s "create 1-1 conversation between non-binding team members (fail)" testCreateOne2OneFailNonBindingTeamMembers,
+      test s "create 1-1 conversation between non-team-members (fail)" testCreateOne2OneFailForNonTeamMembers,
       test s "create 1-1 conversation between binding team members" (testCreateOne2OneWithMembers RoleMember),
       test s "create 1-1 conversation between binding team members as partner" (testCreateOne2OneWithMembers RoleExternalPartner),
-      test s "add new team member" testAddTeamMember,
       test s "poll team-level event queue" testTeamQueue,
-      test s "add new team member binding teams" testAddTeamMemberCheckBound,
       test s "add new team member internal" testAddTeamMemberInternal,
-      test s "remove aka delete team member" testRemoveNonBindingTeamMember,
```

Comment:
I think these should all stay, and use binding teams instead.  (Or have you moved them elsewhere?)

Reply:
No, because "Add Team Member" was only implemented for non-binding teams.
And there is a separate test for removing binding team members.